transactions/src/main/java/org/jboss/as/txn/subsystem/TransactionSubsystemRootResourceDefinition.java

```diff
@@ -378,10 +380,11 @@ protected void finishModelStage(final OperationContext context, final ModelNode
                     ModelNode resourceModel = model.getModel();
                     if (resourceModel.hasDefined(mutualAttributeName) && resourceModel.get(mutualAttributeName).asBoolean()) {
                         resourceModel.get(mutualAttributeName).set(new ModelNode(false));
+                        context.addResponseWarning(Level.WARNING, "Please note that only journal store or jdbc store can be used, thus '" + mutualAttributeName + "' has been disabled.");
```

Comment:
@rpelisse - if you provide neither JDBC nor JOURNAL element then you get a file system store (default) - does that help with the text to use?